src/utils/DecoyDatabase.cpp

```diff
@@ -201,8 +204,64 @@ class TOPPDecoyDatabase :
     Math::RandomShuffler shuffler(seed);
     for (Size i = 0; i < in.size(); ++i)
     {
-      f.readStart(in[i]);
+      // check if decoy_string is common decoy string (e.g. decoy, rev, ...)
+      try
+      {
+        String decoy_string_lower = decoy_string;
+        decoy_string_lower.toLower();
+        bool is_common = 0;
```

Comment:
I always think bool x = 0 evaluates to false. Would be better to read if we init it with : True

Reply:
const auto & ?

Reply:
` = false`
`is_common{false};`